modules/replicator/src/main/java/org/apache/ignite/internal/replicator/exception/PrimaryReplicaMissException.java

```diff
@@ -30,29 +29,27 @@ public class PrimaryReplicaMissException extends IgniteInternalException {
     /**
      * The constructor.
      *
-     * @param requestAddress Primary replica address from request.
-     * @param localAddress Local primary replica address.
      * @param requestTerm Raft term from request.
      * @param localTerm Local raft term.
      */
-    public PrimaryReplicaMissException(NetworkAddress requestAddress, NetworkAddress localAddress, Long requestTerm, Long localTerm) {
-        this(requestAddress, localAddress, requestTerm, localTerm, null);
+    public PrimaryReplicaMissException(Long requestTerm, Long localTerm) {
```

Comment:
Nope, it's still required to know expectedPrimaryReplicaAdress and currentPrimaryReplicaAddress. Expected is the one that was used for routing - in other words recipient localAddress and currentPrimaryReplicaAddress is leader's address that was returned by leaderAndTerm().
By the way, what do u think about renaming params to
expectedPrimaryReplicaAdress
currentPrimaryReplicaAddress
expectedPrimaryReplicaTerm
currentPrimaryReplicaTerm
together with using one-param-on-a-line formatting?

Reply:
Fixed, but expectedPrimaryReplicaAdress and currentPrimaryReplicaAddress are not needed.